core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksBlockStore.java

```diff
@@ -58,8 +57,10 @@
   // This is a field instead of a constant because it depends on the call to RocksDB.loadLibrary().
   private final WriteOptions mDisableWAL;
   private final ReadOptions mIteratorOption;
+  private final ColumnFamilyOptions mCfOpts;
```

Comment:
nit: let's keep naming more expressive
```suggestion
  private final ColumnFamilyOptions mColumnFamilyOptions;
```